index.js

```diff
@@ -144,7 +182,7 @@ server.addMethod("eth_submitWork", (w) => {
       let rate = (solutions / (lastTime - firstTime)) * RATE_DIV;
       console.log(`${(new Date()).toISOString()} | Rate ${rate.toFixed(2)}Mh/s`);
     }
-    State.next();
+    nextOrReorganizeWithProbability(0.1)
```

Comment:
`eth_submitWork` should always change state, since valid work is always advances the chain

Reorgalization is addition to this - 1 or 2 times with probability 33% each should work imho.